StudyBuddy/app/src/main/res/layout/activity_reset_password.xml

```diff
@@ -7,54 +7,38 @@
     tools:context="uw.studybuddy.ResetPassword">
 
     <TextView
-    android:id="@+id/textView"
+    android:id="@+id/textViewReset"
     android:layout_width="368dp"
     android:layout_height="wrap_content"
     android:gravity="center"
-    android:text="Enter your new password"
+    android:text="Please check you email and reset the password"
     android:typeface="serif"
     android:textSize="30dp"
         tools:layout_editor_absoluteY="0dp"
         tools:layout_editor_absoluteX="0dp" />
 
-    <EditText
-        android:id="@+id/resetcode"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:hint="password"
-        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
-        android:textSize="24sp"
-        android:typeface="serif"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView"
-        app:layout_constraintHorizontal_bias="0.0" />
 
     <EditText
-        android:id="@+id/ReeneterCode"
-        android:layout_width="0dp"
+        android:layout_width="368dp"
         android:layout_height="wrap_content"
-        android:hint="retype your Password"
-        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
-        android:textSize="24sp"
         android:typeface="serif"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/resetcode"
-        app:layout_constraintHorizontal_bias="0.0" />
+        android:textSize="30dp"
+        android:id="@+id/ResetEmail"
+        android:layout_marginTop="40dp"
+        app:layout_constraintTop_toBottomOf="@+id/textViewReset"
+        android:hint="Please type your email"
+        tools:layout_editor_absoluteX="8dp" />
 
     <Button
-        android:layout_width="wrap_content"
+        android:layout_width="368dp"
         android:layout_height="wrap_content"
-        android:text="Confirm"
+        android:text="Sent reset Email"
```

Comment:
Can we just have the button title be "Reset Password" or something like this:
![passwordrecovery](https://user-images.githubusercontent.com/17562055/27208915-ab8bb53a-5216-11e7-8e9a-a474fecac5b9.png)